python/tank/deploy/tank_commands/setup_project_wizard.py

```diff
@@ -641,4 +641,4 @@ def execute(self):
             core_localize.do_localize(self._log, 
                                       self._params.get_configuration_location(sys.platform), 
                                       suppress_prompts=True,
-                                      strip_toolkit_credentials=self._is_session_based_authentication_supported())
```

Comment:
No worries (sorry I've been out of touch this week).

I do think we should get rid of it... I don't think we'll be doing that level of configuration in the wizard, but if we do it is easy enough to add back in.  That is better than having it around now and having to support it forever more.